### PR TITLE
Improve template usage instructions

### LINKED ISSUE #124: gh-pages branch does not transfer when following current README instructions
I just set up a page for a course for the first time. It mostly went painlessly (I'm pretty impressed with this template! 🎉), but I had to deviate from the instructions in the README somewhat:

Step 4 says:
> Make sure the repository is public, leave "Include all branches" unchecked, and click on "Create repository from template". You will be redirected to your new copy of the workshop template repository.

However, when you do this, the gh-pages branch does not transfer to the new repo, so it cannot be used to make the page.

I ended up ticking the box to include all branches, then manually deleting the branches except for gh-pages. This had the desired effect, but of course was a bit cumbersome. I couldn't find an easier way to do it, but feel like there must be one?

(Also a small point: in Step 1 the address of this repository is still referred to as https://github.com/esciencecenter-digital-skills/next-workshop-template -- it should be https://github.com/esciencecenter-digital-skills/workshop-template).

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,9 @@ please [submit an issue][issues] or [mail us][email].
     `YYYY-MM-DD` is the start date of the workshop, `ds` stands for Digital
     Skills, and `gpu` is the workshop name in this example.
 
-4.  Make sure the repository is public, leave "Include all branches" unchecked,
-    and click on "Create repository from template". You will be redirected to
+4.  Make sure the repository is public, leave "Include all branches" unchecked 
+    (don't worry about the `gh-pages` branch, it will be created by a github action).
```
Comment (bvreede) [inline README.md:29]: Aha! That I did not realize when I went through this workflow. 
Perhaps it is good then to add a step either between 4 and 5, or as part of 5, to verify that the actions have completed, and that the gh-pages branch is created? Or warn the user that it does take a little while; when I tried it out just now this Action took 5 minutes to complete.

**Review by bvreede — CHANGES_REQUESTED** [review]
Thanks for addressing my issue! This makes things clear, I had completely missed that this happens. Small suggestion for a little more clarification inside :)

File: README.md (hunk 1 above)
Comment (svenvanderburg) [reply README.md:29]: Yes, that is important info.

Comment (svenvanderburg) [reply README.md:29]: Added.

**Comment by svenvanderburg** [comment]
Thanks @bvreede :)

**Pull request merged**